tests/DMSDocumentTest.php

```diff
@@ -65,12 +67,24 @@ public function testDocumentHasCmsFieldForManagingRelatedDocuments()
     }
 
     /**
+     * Ensures that the DMS Document CMS Related and Versions fields are removed if user can't edit
+     */
+    public function testDocumentHasNoCMSFieldsForManagingRelatedDocumentsIfCantEdit()
+    {
+        $this->logInWithPermission('another-user');
+        $document = $this->objFromFixture('DMSDocument', 'doc-only-these-users');
+        $gridField = $this->getGridFieldFromDocument($document);
+        $this->assertNull($gridField);
```

Comment:
This is misleading because of the way the existing method `getGridFieldFromDocument` works - it's returning the related documents GridField but it doesn't say that in its name (my fault probably). Could you make this a bit more verbose so it's obvious that you're asserting here that the related documents GridField is null?